packages/roger/watch.ts

```diff
@@ -68,7 +68,12 @@ export default async function (port = 9160): Promise<void> {
           // get containing dir of the trio file
           const parentDir = parse(path).dir;
           const contents = await fs.readFile(path, "utf8");
-          const { substance, style, domain } = JSON.parse(contents);
+          const pathContents = JSON.parse(contents);
+          const { style, domain, substance } = pathContents;
+          let { excludeWarnings } = pathContents;
+          if (excludeWarnings === undefined) {
+            excludeWarnings = [];
+          }
```

Comment:
```suggestion
          const { style, domain, substance, excludeWarnings } = JSON.parse(contents);        
```

For brevity mostly, also see below

Reply:
done